style.css

```diff
@@ -0,0 +1,236 @@
+    width: 100%;
+    max-width: 600px;
+    text-align: center;
+    z-index: 1;
+  }
+
+  .toolbar-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+  }
+
+  .menu-items {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: #fff;
+    color: #3a4a42;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    display: none;
+    padding: 5%;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    font-size: 5vw;
+    font-weight: 600;
+    line-height: 10vw;
+  }
+
+  .menu li {
+    display: block;
+    margin: 5% 0;
+    list-style: none;
+  }
+
+  .hamburger {
+    order: 2;
+    margin-left: auto;
+    cursor: pointer;
+    font-size: 6vw;
+    margin-right: 5%;
+  }
+
+  .menu li:first-child {
+    margin-top: 0;
+  }
+
+  .menu li a {
+    padding: 5%;
+    display: block;
+    font-weight: bold;
+    text-decoration: none;
+  }
+
+  /* Show menu items on hover */
+  .toolbar-menu:hover .menu-items {
+    display: block;
+  }
+
+  .menu-items ul {
+    list-style: none;
+    padding-left: 0;
+  }
+
+  .menu-items::after {
+    content: "\f00d";
+    font-family: 'Font Awesome 5 Free', sans-serif;
+    font-weight: 900;
+    font-size: 6vw;
+    position: absolute;
+    bottom: -20px;
+    right: 10%;
+    cursor: pointer;
+    z-index: 2;
+  }
+
+  .social-icons {
+    display: flex;
+    flex-direction: colum;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .social-icons li:last-child {
+    margin-right: 0;
+  }
+
+  .chevron-icon {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 8px;
+    position: absolute;
+    left: calc(50% - 40px / 2 + 0.5px);
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .header {
+    padding: 60px;
+    background-image: url(./assets/images/Header-llustration-desktop.png);
+    background-size: cover;
+    background-position: center;
+  }
+}
+
+.social-icons li {
+  margin-right: 20px;
+}
+
+.menu-items li a {
+  color: #000;
+  text-decoration: none;
+  display: block;
+  padding: 10px;
+  font-weight: bold;
+}
+
+.menu {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menu ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menu li {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.menu li:first-child {
+  margin-left: 0;
+}
+
+.menu li a:hover {
+  background-color: #70c572;
+  color: #fff;
+  text-decoration: none;
+}
+
+.menu-items {
+  display: none;
+}
+
+/* hambuger icon */
+.hamburger {
+  position: absolute;
+  align-items: center;
+  justify-content: flex-end;
+  width: 40px;
+  height: 40px;
+  top: 20px;
+  right: 20px;
+}
+
+/* toolbar  Styles */
+.toolbar {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+}
+
+.header-content h1:nth-of-type(2) {
+  color: #36b37e;
+}
+
+.header-content p {
+  font-size: 8px;
```

Comment:
- [ ] this is how the figma headline text is set

font-family: Inter;
font-size: 20px;
font-weight: 400;
line-height: 28px;
letter-spacing: 0em;
text-align: center;